luigi/tools/migrate.py

```diff
@@ -0,0 +1,133 @@
+"""
```

Comment:
Yes, I can make an automatic check on luigid startup.  The options are either migrate automatically or exit with message to run luigi-migrate.